src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -99,6 +98,15 @@ public RuntimeMetrics(RuntimeMetricsOptions options)
             {
                 this.meter.CreateObservableCounter($"{metricPrefix}assembly.count", () => (long)AppDomain.CurrentDomain.GetAssemblies().Length, description: "Number of Assemblies Loaded.");
             }
+
+            if (options.IsExceptionsEnabled)
```

Comment:
This sounds like broken English, either "is exception" or "are exceptions".

Reply:
Renamed to `ExceptionCountEnabled` and `IsExceptionCountEnabled`